install_config/install/stand_alone_registry.adoc

```diff
@@ -125,35 +125,75 @@ ifdef::openshift-enterprise[]
 [[registry-quick-installation]]
 === Quick Installation for Stand-alone OpenShift Container Registry
 
-When using the quick installation method to install stand-alone OCR, start the
+. When using the quick installation method to install stand-alone OCR, start the
 interactive installation by running:
-
++
 ----
 $ atomic-openshift-installer install
 ----
 
-Then follow the on-screen instructions to install a new registry. The installation questions will be largely the same as if you were installing a full {product-title} PaaS, but when you reach the following screen:
-
-====
+. Then follow the on-screen instructions to install a new registry. The installation questions will be largely the same as if you were installing a full {product-title} PaaS. When you reach the following screen, be sure to choose `2` to follow the registry installation path.:
++
```

Comment:
`Then follow the on-screen instructions to install a new registry.`
Seems a bit obvious at this point. ;-)